src/layout/components/modals/ConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,29 @@
+import Button from "react-bootstrap/esm/Button";
+import Modal from "react-bootstrap/esm/Modal";
+
+export const ConfirmDeleteModal = ({
```

Comment:
To jest komponent Bootstrap ale napisany w Typescript zamiast HTML.

![image](https://github.com/pawelNu/compstore-ui/assets/93542936/894ca738-a353-45df-a774-f55ad58b8305)

Wszystkie listy rozwijane, checkboxy, przyciski też można tak zrobić.

https://react-bootstrap.netlify.app/docs/getting-started/introduction

Jak chcesz to możemy nad tym pomyśleć, aby tak pisać.

Reply:
To jest ta kwestia używania gotowych komponentów zamiast pisanych samodzielnie? Jestem za